iOS-Weather-App/Controllers/WeatherViewController/WeatherViewControllerModel.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  WeatherViewControllerModel.swift
+//  iOS-Weather-App
+//
+//  Created by pavels.garklavs on 22/02/2022.
+//
+
+import MapKit
+import CoreLocation
+import Foundation
+
+final class WeatherViewControllerModel {
+    var onSearchTapped: (() -> Void)?
+    var onTableButton: (() -> Void)?
+
+    func celsius(_ kelvin: Double?) -> Double {
```

Comment:
when we make a request, we can set metric system and get temperature in celsius, also think, that this logic should be setting at model